packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useModifiersStore.js

```diff
@@ -72,7 +72,7 @@ export function useModifiersStore() {
    * @param {string} name - The name of the modifier to toggle
    */
   const baseHandleToggle = (name) => {
-    if (modifiers[name]) {
+    if (modifiers && modifiers[name]) {
```

Comment:
why would you have no modifiers?

Reply:
Slate [always returns null](https://github.com/ianstormtaylor/slate/blob/main/packages/slate/src/editor/marks.ts#L11-L13) if there's no selection in Editor.marks

Reply:
oh wow, weird API name but thanks!